Wikipedia/Code/ReusableCell.swift

```diff
@@ -0,0 +1,11 @@
+import UIKit
+
+protocol ReusableCell {
+    static var reuseIdentifier: String { get }
+}
+
+extension ReusableCell {
+    static var reuseIdentifier: String {
```

Comment:
Just an FYI, this felt familiar so I did some digging - we do have something similar in UIView+Identifier.swift for views and cells. But I do like that this feels both more specific to cells and more flexible as a protocol, so I'm open to keeping this here.

Reply:
Ah I missed this extension. Thanks for pointing it out! I have a minor quibble with the naming of it - `identifier` to me indicates something unique, whereas `reuseIdentifier` at least indicates it's not something globally unique, intended to be reused. Because you also see the value in this flexible protocol/specific to cell approach, I'd rather we eventually use _this_ extension for the current instances using the `UIView` one rather than using that one here.